### PR TITLE
Release version 2.1.6

### PR DESCRIPTION
**Pull request by stel**
Version 2.1.6 with Realm 2.4.3

/cc @jpsim

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2.1.6 Release notes (2017-03-02)
+=============================================================
+* Updated to Realm 2.4.3
```
Comment (bdash) [inline CHANGELOG.md]: It might be worth calling out that this fixes an issue with token expiration for Realm's opened from Realm Mobile Platform.

Comment (stel) [inline CHANGELOG.md]: 👌

**Comment by stel** [comment]
Submitted to the Review.

**Comment by stel** [comment]
Available on MAS, merging.

**Pull request merged**